Macro definition is not working

```
$ cin
cin - v1.2.0
>> #define P 256u

>> int x = P;
/tmp/cin/tmp_code.c: In function ‘main’:
/tmp/cin/tmp_code.c:2:9: error: ‘P’ undeclared (first use in this function)
 int x = P;
         ^
/tmp/cin/tmp_code.c:2:9: note: each undeclared identifier is reported only once for each function it appears in

>>
```

--- a/src/files.c
+++ b/src/files.c
@@ -15,12 +15,20 @@ void write_to_file() {
     if (file > 0) {
         code_history *it;
         include_history *iit;
+        macro_history *mit;
 
         iit = include_start;
         while (iit) {
             write(file, iit->buffer, iit->length);
 
             iit = iit->next;
+        }
+
+        mit = macro_start;
+        while (mit) {
+            write(file, mit->buffer, mit->length);
+
+            mit = mit->next;
         }
 
         it = start;

